### DISCUSSION TITLE
Downloading only the most recent medias and then stops at the first empty timeline cursor

### DISCUSSION THREAD

**Discussion by Okarodb** [Avnsx/fansly#152] [Q&A]
Hello,

I have a problem downloading a particular account, to which I subscribed today. It has more than 800 pics but Fansly-Downloader (0.4.1) only downloads 17 of them, available in the most recent timeline cursor. The following timeline cursor has 0 media, and the software finishes, although it clearly indicates 857 total pictures for the creator, of which I can view the vast majority.

Is this normal ? What may I do to get around this ?

Thanks in advance.

**Comment by Avnsx** [comment]
https://github.com/Avnsx/fansly-downloader/issues/148

**Comment by Okarodb** [comment]
Hadn't noticed that issue discussion, sorry. I'll be waiting for version 0.4.2. Thank you very much !
